games/minetest/Portfile

```diff
@@ -85,20 +103,22 @@ patchfiles-append       003-patch-ignore-psn-option-mac-bundle.diff
 
 configure.args-append   -DCMAKE_BUILD_TYPE=Release \
                         -DCMAKE_INSTALL_PREFIX:PATH=${applications_dir} \
-                        -DBUILD_CLIENT=1 \
-                        -DENABLE_SOUND=1 \
-                        -DENABLE_REDIS=1 \
-                        -DENABLE_POSTGRESQL:BOOL=OFF \
-                        -DENABLE_LEVELDB=1 \
-                        -DENABLE_FREETYPE=1 \
-                        -DENABLE_CURL=1 \
-                        -DENABLE_GETTEXT=1 \
-                        -DENABLE_SYSTEM_JSONCPP=1 \
-                        -DENABLE_UPDATE_CHECKER=0 \
```

Comment:
we do want to disable the internal update checker - this should go through MacPorts

Reply:
OK - a mishap from my side

Yes, I have to admit I overlooked this but did not want to revert by another commit without anyone complaining.

```
-DENABLE_UPDATE_CHECKER=OFF
```

You were not complaining but giving **reasonable advice** to be clear on that please.